package.json

```diff
@@ -20,6 +20,8 @@
     "@ethersproject/strings": "^5.4.0",
     "@ethersproject/units": "^5.4.0",
     "@ethersproject/wallet": "^5.4.0",
+    "@gnosis.pm/safe-apps-provider": "^0.8.0",
+    "@gnosis.pm/safe-apps-sdk": "^4.3.0",
```

Comment:
Don't these belong to the lock repo? They are used [here](https://github.com/snapshot-labs/lock/blob/master/connectors/gnosis.ts) but not included in [package.json](https://github.com/snapshot-labs/lock/blob/master/package.json).

Reply:
We aren't adding the dependencies on Lock.js directly, I think we do this to get more flexibility, but if we think its stable we could move all dependencies back to Lock